tests/tnimterop_c.nim

```diff
@@ -10,7 +12,7 @@ cDefine("FORCE")
 cIncludeDir testsIncludeDir()
 cCompile cSearchPath("test.c")
 
-cPluginPath("tests/tnimterop_c_plugin.nim")
+cPluginPath(getProjectPath() / "tnimterop_c_plugin.nim")
```

Comment:
This was needed to pass tests. For some reason the directory isn't correct here in docgen to just use a relative path